### PR TITLE
Issue #2632: Core fieldable file entity bundles

### PR DESCRIPTION
**Pull request by Al-Rozhkov**

### CODE REVIEW

**Comment by Al-Rozhkov** [comment]
Closing to restart sandbox

**Comment by Al-Rozhkov** [comment]
Closing and reopen to fix the problem with missed Alt and Title fileds on Image bundle.

File: core/modules/file/file.file.inc [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  // Add support for mka and mkv.
+  // @todo Remove when http://drupal.org/node/1443070 is fixed in core.
+  $new_mappings['mka'] = 'audio/x-matroska';
+  $new_mappings['mkv'] = 'video/x-matroska';
+
+  // Add support for weba, webm, and webp.
+  // @todo Remove when http://drupal.org/node/1443070 is fixed in core.
+  $new_mappings['weba'] = 'audio/webm';
+  $new_mappings['webm'] = 'video/webm';
+  $new_mappings['webp'] = 'image/webp';
```
Comment (quicksketch) [inline core/modules/file/file.file.inc:146]: Looks like these mappings can be deleted. See d3b960e15895c5f448cb3c1b4c5d4981ab69976f.

File: core/modules/file/file.install [hunk 2]
```diff
@@ -105,11 +105,23 @@ function file_schema() {
         'not null' => TRUE,
         'default' => 0,
       ),
+	  'type' => array(
```
Comment (quicksketch) [inline core/modules/file/file.install:108]: Minor spacing issue, remove indentation.

**Review by quicksketch — CHANGES_REQUESTED** [review]
Minor issues in my first read-through. I think we may want to reconsider the "file_metadata" table, which for core purposes really just means image width/height. I'll post more feedback to the parent issue at https://github.com/backdrop/backdrop-issues/issues/2632.

**Pull request closed without merging**